src/hooks/useBulkPermissions.js

```diff
@@ -32,12 +32,21 @@ export const useBulkPermissions = () => {
   const isActionMenuShown = hasCsvEditPerms || hasAnyInAppEditPermissions;
   const isSelectIdentifiersDisabled = !hasAnyEditPermissions;
   const isDropZoneDisabled = !hasAnyEditPermissions;
-  const isInventoryRadioDisabled = !hasInAppViewPerms;
   const isUserRadioDisabled = !hasCsvViewPerms && !hasInAppUsersEditPerms;
   const hasOnlyInAppViewPerms = hasInAppViewPerms && !hasCsvEditPerms && !hasInAppEditPerms && !hasInAppUsersEditPerms;
+  const hasItemInventoryView = hasInAppViewPerms && hasItemsPerms;
+  const hasHoldingsInventoryView = hasInAppViewPerms && hasHoldingsPerms;
+  const hasItemsAndHoldingsInventoryView = hasInAppViewPerms && hasInventoryInstanceViewPerms;
+  const hasItemInventoryEdit = hasInAppEditPerms && hasItemsPerms;
+  const hasHoldingsInventoryEdit = hasInAppEditPerms && hasHoldingsPerms;
+  const hasAnyInventoryWithInAppView = (hasItemInventoryView && hasInAppViewPerms)
+      || (hasHoldingsInventoryView && hasInAppViewPerms)
+      || (hasItemsAndHoldingsInventoryView && hasInAppViewPerms);
```

Comment:
This approach looks shorter

```suggestion
  const hasAnyInventoryWithInAppView = (hasItemInventoryView  || hasHoldingsInventoryView || hasItemsAndHoldingsInventoryView) && hasInAppViewPerms);
```

Reply:
Thanks @artem-blazhko , it's really look better